### PR TITLE
KRACOEUS-8738 fixing landing page text

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml [hunk 1]
```diff
@@ -37,9 +37,19 @@
...
+                      	use for casual or infrequent users of the system while retaining the full set of powerful features from previous KC releases."/>
+                <bean parent="Uif-Message" p:messageText="Specific Features include:"/>
+                <bean parent="Uif-Message" p:messageText="* Re-organized and intuitive data entry."/>
+                <bean parent="Uif-Message" p:messageText="* Left hand navigation and 'Save and Continue' options for linear navigation through the system. No more 'where do I go next'?"/>
+                <bean parent="Uif-Message" p:messageText="* Quick access to tools: validation, print, copy, medusa, hierarchy management, and budget version management is always one-click away in a Proposal Development Document."/>
+                <bean parent="Uif-Message" p:messageText="* Search while you type for several frequently entered data fields."/>
+                <bean parent="Uif-Message" p:messageText="* Modal-based user interactions that provide context and continuity while performing lookups and entering large amounts of data."/>
```
Comment (blackcathacker) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml]: I feel like this should be an unordered list instead. Uif-SimpleListGroup or Uif-ListSection or something like that.

Comment (jhulslander) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml]: I wanted an unordered list but I couldn't find that if component in the sample app tool. I will give this a go.

Comment (gathreya) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml]: Is this really what that text is supposed to say?  Several designed????

Comment (tdurkin) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml]: Harsh! I wrote that up pretty quickly. It's not gospel and we needed to get rid of the lorem ipsum. Ok with me if we tweak it some more.

Sent from my iPhone

On Jan 22, 2015, at 10:07 PM, Gayathri Athreya <notifications@github.com<mailto:notifications@github.com>> wrote:


In coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml<https://github.com/kuali/kc/pull/1511#discussion_r23429495>:

>                  <bean parent="Uif-Message"
> -                      p:messageText="Lorem ipsum dolor sit amet, conse ctetuer adipiscing elit, sed diam nonu mmy nibh euismodt. Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet, conse ctetuer adipiscing elit, sed diam nonu mmy nibh euismodt. Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet."/>
> +                      p:messageText="The process for developing a Proposal and accompanying Budget has been streamlined. The design emphasizes ease of
> +                             use for casual or infrequent users of the system while retaining the full set of powerful features from previous KC releases."/>
> +                <bean parent="Uif-Message" p:messageText="Specific Features include:"/>
> +                <bean parent="Uif-Message" p:messageText="* Re-organized and intuitive data entry."/>
> +                <bean parent="Uif-Message" p:messageText="* Left hand navigation and 'Save and Continue' options for linear navigation through the system. No more 'where do I go next'?"/>
> +                <bean parent="Uif-Message" p:messageText="* Quick access to tools: validation, print, copy, medusa, hierarchy management, and budget version management is always one-click away in a Proposal Development Document."/>
> +                <bean parent="Uif-Message" p:messageText="* Search while you type for several frequently entered data fields."/>
> +                <bean parent="Uif-Message" p:messageText="* Modal-based user interactions that provide context and continuity while performing lookups and entering large amounts of data."/>


Is this really what that text is supposed to say? Several designed????

Comment (gathreya) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml]: Oops! Wanted to make sure he got it right.  I am sure my English is  not as good as a native English speaker's so it probably reads OK to others.

Comment (gathreya) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml]: Kuali Coeus 6.0 is the first of its kind designed to improve user experience. ???

Comment (gathreya) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/krad/common/LandingPageView.xml]: Or even "Kuali Coeus 6.0 is the first of several releases designed to improve user experience."

**Comment by jhulslander** [comment]
This is ready for review again
I fixed the list format per @blackcathacker 
I changed @tdurkin 's wording per @gathreya 's suggestion

**Pull request merged**